server/routes/api/place.py

```diff
@@ -52,15 +53,25 @@ def child_fetch(dcid):
       post=True
     )
     places = response[dcid].get('in', [])
+    dcid_str = '^'.join(sorted(map(lambda x: x['dcid'], places)))
+    pop = json.loads(get_stats_wrapper(dcid_str, 'TotalPopulation'))
```

Comment:
the flask cache module can only take a string as key, if passing in an array, it will complain. I think we can improve this by preloading the population for all places at server start up?

Reply:
Oh! Yea, preloading the populations would be great..